Lickability/PinpointKit
# Remove Swiftlint As Framework / Add As Pod

Closes #225 

## What It Does
Switches to using Swiftlint as a pod.

## How to Test
Make sure the project runs and there are no warnings.
Conversely, make sure Swiftlint warnings are appropriately generated for both the example and the framework.
